JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/GroupBean.java

```diff
@@ -8,132 +11,119 @@
 import java.util.LinkedList;
 import java.util.Map;
 import java.util.HashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
-import ru.javafiddle.jpa.entity.Access;
-import ru.javafiddle.jpa.entity.UserGroup;
-import ru.javafiddle.jpa.entity.Group;
-import ru.javafiddle.jpa.entity.User;
+import ru.javafiddle.jpa.entity.*;
 
 
 @Stateless
 
 public class GroupBean {
 
-    @PersistenceContext
+    private static final Logger logger =
+            Logger.getLogger(ProjectBean.class.getName());
+    @PersistenceContext(name = "JFPersistenceUnit")
     EntityManager em;
 
-    public GroupBean() {
-    }
+    public GroupBean(){}
+
+//we always know that the group were we are trying to add a user exits now
+
+//CALL THIS METHOD WHEN THE PERSON IS CREATING A GROUP BY HIMSELF!!
 
-    //we should get at first userId, then add this element to UserGroups
-    //when we are in this method we definitly have group with such id
-    public void addMember(int groupId, String userNickName, String accessRights) {
+    @TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
+    public void addMember(int groupId, String groupName,String userNickName, String accessRights) {
```

Comment:
Bad class design. You should avoid building POJO in EJB's because such operations do not require EJB context.
You'd better have something like void addMember(Group, User);